### DISCUSSION TITLE
Open a document with "preview" tab

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#10088] [General]
If we open a document in PIMCore, it will open with the "Edit" tab by default. But I like to open in "Preview" tab by default. Please help me.

eg: pimcore.helpers.openElement(2, 'document');

The above code will open a document but I need to open the document via Extjs or JS and it should land on the "preview" tab of the document.

**Comment by weisswurstkanone** [comment]
An option would maybe be to attach to this event and then set the active tab accordingly.

See 
https://github.com/pimcore/pimcore/blob/07357962b0ba5bd1e141a377461caed5c962663e/bundles/AdminBundle/Resources/public/js/pimcore/document/page_snippet.js#L68

and

https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Plugin_Backend_UI.html
